static/scripts/filter-glossary.js

```diff
@@ -0,0 +1,23 @@
+function addFilterEvent() {
```

Comment:
This should work if you make this into a component here: https://github.com/temporalio/documentation/tree/main/docs/components

Then import it into the glossary.md file the same we were importing Tabs for example.